src/voicefocus/VoiceFocusDeviceTransformer.ts

```diff
@@ -301,8 +310,8 @@ export class VoiceFocusDeviceTransformer {
     preload: boolean
   ): Promise<[VoiceFocus, VoiceFocusTransformDeviceDelegate]> {
     const delegate = new VoiceFocusTransformDeviceDelegate();
-    const vf = await VoiceFocus.init(config, { delegate, preload, logger: this.vfLogger });
-    return [vf, delegate];
+    this.vf = await VoiceFocus.init(config, { delegate, preload, logger: this.vfLogger });
```

Comment:
If someone calls this internal method consecutively, could there be any memory leaks since subsequent calls merely reassign a new VF?

Reply:
Yes, but technically the create() funciton inside VoiceFocusDeviceTransformer is a static function, so someone should be able to create multiple transformers already - that is not a bug, that is how it was originally implemented. The piece we were missing is a static destroy() function, which I added in this PR.

Now, at least developers can tie up any loose memory leaks that they might've opened if they created multiple VoiceFocusDeviceTransformers.